src/components/layout/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import classes from './Layout.module.css'
+import NavBar from "./NavBar";
+import React from "react";
+
+/**
+ * Generic layout component to ensure consistent styling of the app.
+ * @param props any child component to be passed through
+ * @constructor
```

Comment:
Since we are not dealing with class based components, is this the appropriate documentation? Is this suggested as a convention somewhere?

Reply:
Ah so this was added by default by my constructor but I had a look into it and I think it helps typescript do a better job https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#constructor happy to remove if you think it's clunky though